### ISSUE TITLE
A11y_ML .NET Model Builder_ Data_ High Contrast: Selected Radio button text is not visible in high contrast black and high contrast white mode

### ISSUE THREAD

**Issue by SaisreeDandamudi** [dotnet/machinelearning-modelbuilder#1130]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #Feature-ML.NETModelBuilder-Win32-Nov2020; #A11ySev1; #Benchmark; #Visual Studio; #HighContrast; #WCAG4.3.1;

### Environment Details:
Microsoft Visual Studio 2019preview Version 16.8.0 Preview 6.0

### Note:
Same issue is observed in "Advanced Data Options" windows. 

### Repro Steps:

1. Enable High contrast black mode in the system
2. Open Visual Studio Enterprise 2019 Preview 
3. Create a new .Net Core Console App (C#) 
4. Tab to "Solution Explorer" and right click on the project (Shift + F10) and then from the expanded list Select "Add" and then from the list items select "Machine learning"
5. Tab till "Text Classification" scenario and select the scenario
6. Tab till "Next" button and press enter
7. Verify whether all the content and controls are clearly visible in high contrast mode

### Actual Result:
Selected Radio button text is not visible in high contrast black and high contrast white mode

### Expected Result:
Selected radio button text should be clearly visible in high contrast mode as well

### User Impact:
Users who use the application in high contrast mode will miss the content if all the content is not clearly visible in high contrast mode

### Attachment for Reference
[1130_ML .NET Model Builder_ high contrast.pptx](https://github.com/dotnet/machinelearning-modelbuilder/files/5630123/1130_ML.NET.Model.Builder_.high.contrast.pptx)

**Comment by SaisreeDandamudi** [comment]
Verified the bug in latest build and the issue is fixed. Hence closing the bug
Please find the attachment below for reference
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.9.0 Preview 4.0 [30917.4.main]
![Screenshot (59)](https://user-images.githubusercontent.com/51435580/108043609-4ebec680-7067-11eb-8f45-610022600d0e.png)
![Screenshot (61)](https://user-images.githubusercontent.com/51435580/108043622-52524d80-7067-11eb-991d-113fa72c58f6.png)
